packages/app/src/systems/Accounts/hooks/useAccountConnectionFuel.tsx

```diff
@@ -0,0 +1,29 @@
+import {
+  useFuel,
+  useConnect,
+  useAccount,
+  useDisconnect,
+} from '@fuels-portal/hooks';
+
+import { store } from '~/store';
+
+export const useAccountConnectionFuel = () => {
+  const fuel = useFuel();
+  const { address, isConnected } = useAccount();
+  const { connect, error, isLoading } = useConnect();
+  const { disconnect } = useDisconnect();
+
+  return {
+    handlers: {
+      connect,
+      disconnect,
+      openFuelInstall: store.openFuelInstall,
+      closeDialog: store.closeOverlay,
+    },
+    fuel,
```

Comment:
we should not return `fuel`, could return something like:
`hasInstalledFuel: Boolean(fuel)`